src/App/Exercises/Js/EmptyValuesAndComments/EmptyValuesAndComments.jsx

```diff
@@ -54,8 +54,8 @@ export const EmptyValuesAndComments = () => {
     setFlag(0);
   }
   function replacePixelsToMegaPixels() {
-    if (flag === 0 && String(pixels).length >= 4) {
-      setPixels(Number(pixels) / 1000);
+    if (flag === 0 && String(pixels).length >= 6) {
+      setPixels(Number(pixels) / 1000000);
```

Comment:
Racja 😃 wcześniej były KiloPixele xD

Reply:
O tak, i w ogóle była to zmiana w stylu : 
Odchodzę od komputera mija 20 min a mnie oświeca " Kurde tysiąc to przecież Kilo !"